transport/event_notifier.cc

```diff
@@ -43,11 +44,15 @@ void cql_server::event_notifier::on_create_keyspace(const sstring& ks_name)
     for (auto&& conn : _schema_change_listeners) {
         using namespace cql_transport;
         if (!conn->_pending_requests_gate.is_closed()) {
+          try {
             conn->write_response(conn->make_schema_change_event(event::schema_change{
                 event::schema_change::change_type::CREATED,
                 event::schema_change::target_type::KEYSPACE,
                 ks_name
             }));
+          } catch (...) {
+            elogger.warn("Create keyspace notification failed {}: {}. Ignored", ks_name, std::current_exception());
```

Comment:
I'm not even sure how to drop it.

Reply:
I see that we have `conn->shutdown()`.
But I suggest we do this as a followup since it may risk this PR that is a step forward in the right direction.

Reply:
done in v4